.scripts/util.sh

```diff
@@ -59,25 +63,73 @@ is_generated() {
     fi
 }
 
+select_browser() {
+    local browser_engine="$1"
+
+    if [ "$browser_engine" = "Chromium" ]; then
+        BROWSER="Chromium"
+        BROWSER_INCLUDE_DIR="$CHROMIUM_INCLUDE_DIR"
+        BROWSER_LIB_DIR="$CHROMIUM_LIB_DIR"
+        SYS_OPTS="-DCHROMIUM_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DCHROMIUM_LIB_DIR=$BROWSER_LIB_DIR"
```

Comment:
We want to store the sourced environment globals for the whole terminal session, so: `BROWSER=$1`

It will ulitimately be much more maintainable code if we pass to cmake:
- -DBROWSER=$BROWSER
- -DBROWSER_INCLUDE_DIR="$..."
- -DBROWSER_LIB_DIR="$..."

and then process logic within cmake. There can only ever be ONE browser definition, so try not to spawn a large amount of variable names at the highest level, they will only need to be dealt with at lower levels. Think 3 steps ahead and look for maximum direct simplicity.